libs/sdk-ui-dashboard/src/presentation/widget/widget/DefaultDashboardInsightWidget.tsx

```diff
@@ -146,6 +149,8 @@ const DefaultDashboardInsightWidgetCore: React.FC<
             screen={screen}
         >
             <DashboardItemVisualization
+                isSelectable={isEditMode}
+                objRef={insight.insight.ref}
```

Comment:
This is not going to work well, what if there are two widgets with the same insight?